lib/interface.rb

```diff
@@ -0,0 +1,96 @@
+class Interface
+
+  attr_reader :menu_array, :order
+
+  def initialize(filepath)
+    @menu = Menu.new(filepath)
+    @order = Order.new
+    @order_array = []
+    @menu.build_menu_display
+  end
+
+  def user_interaction
+    welcome
+    build_order(@menu, @order)
+    show_order
+    order_confirm
+  end
+
+  def welcome
+    file_reader('/Users/malinnaleach/Programs/takeaway-challenge/lib/welcome.txt')
+    display(@menu.menu_array)
+  end
+
+  def file_reader(filepath)
+    File.open(filepath) {|f| puts f.read}
+  end
+
+  def display(array)
+    array.each {|string| puts string}
+  end
+
+  def build_order(menu, order)
```

Comment:
Could this method be refactored to use private helper methods to make it shorter and clearer?